bughouse_console/src/rusqlite_server_hooks.rs

```diff
@@ -0,0 +1,183 @@
+use bughouse_chess::event::*;
+use bughouse_chess::server::*;
+use bughouse_chess::server_hooks::ServerHooks;
+use bughouse_chess::*;
+use log::error;
+
+pub struct RusqliteServerHooks {
+    invocation_id: String,
+    game_number: u64,
+    game_start_time: Option<std::time::SystemTime>,
+    conn: rusqlite::Connection,
+}
+
+impl RusqliteServerHooks {
+    pub fn new(address: &str) -> Result<Self, Box<dyn std::error::Error>> {
+        let conn = rusqlite::Connection::open(address)?;
+        conn.execute(
+            "CREATE TABLE IF NOT EXISTS finished_games (
+                    invocation_id TEXT NOT NULL,
+                    game_number INTEGER NOT NULL,
```

Comment:
Do you know if it's possible to make an auto-incrementing integer key instead of the (invocation_id, game_number) pair? (To make addressing individual games easier.) If not, it's ok.

Reply:
Apparently, sqlite adds a `rowid` which auto-incerements, if no `PRIMARY KEY` is specified. Keeping invocatin uuid for now to distinguish between server restarts, let me know if you want me to remove that as well.

Reply:
Invocatin uuid sounds ok. We could also add build information from git in the future (no need to do this now, just thinking out loud).